src/screens/PinScreen.js

```diff
@@ -116,13 +119,15 @@ class PinScreen extends React.Component {
        * }
        */
       const credentials = await Keychain.getGenericPassword({
-        authenticationPrompt: this.biometryText,
+        authenticationPrompt: { title: this.biometryText },
```

Comment:
Passing a simple string on this parameter is now deprecated. Changed to the most up-to-date method signature.